### PR TITLE
Cu-860qqrfeg daemon proxy support

### PR DESCRIPTION
**Pull request by michaeldeift**
public: Support proxies in daemonset

### CODE REVIEW

File: charts/k8s-watcher/templates/configmap-daemon.yaml [hunk 1]
```diff
@@ -19,6 +19,11 @@ data:
     [[outputs.http]]
       url = "https://app.komodor.com/metrics-collector/api/v1/collect"
       content_encoding = "gzip"
+      {{- if .Values.proxy.https }}
+      http_proxy_url = "${KOMOKW_HTTPS_PROXY_URL}"
+      {{- else if .Values.proxy.http }}
+      http_proxy_url = "${KOMOKW_HTTP_PROXY_URL}"
+      {{- end }}
```
Comment (Noy-Simon) [inline charts/k8s-watcher/templates/configmap-daemon.yaml:26]: did you check it ? 
i dont think it works inside a string value

File: charts/k8s-watcher/templates/daemonset.yaml [hunk 2]
```diff
@@ -48,6 +48,7 @@ spec:
                   name: {{ include "k8s-watcher.name" . }}-secret
                   key: apiKey
                   {{- end }}
+{{- include "k8s-watcher.proxy-conf" . }}
```
Comment (Noy-Simon) [inline charts/k8s-watcher/templates/daemonset.yaml:51]: need an end tag no?

Comment (michaeldeift) [reply charts/k8s-watcher/templates/daemonset.yaml:51]: no..it's the same in agent's file

File: charts/k8s-watcher/templates/configmap-daemon.yaml (hunk 1 above)
Comment (michaeldeift) [reply charts/k8s-watcher/templates/configmap-daemon.yaml:26]: Yes..it works..the whole file is defined like this

Comment (Noy-Simon) [reply charts/k8s-watcher/templates/configmap-daemon.yaml:26]: even the ifs ?

Comment (michaeldeift) [reply charts/k8s-watcher/templates/configmap-daemon.yaml:26]: yes...tested with --set proxy.http="something" and --set proxy.https="sdad"

File: charts/k8s-watcher/templates/configmap-daemon.yaml [hunk 3]
```diff
@@ -19,6 +19,11 @@ data:
     [[outputs.http]]
       url = "https://app.komodor.com/metrics-collector/api/v1/collect"
       content_encoding = "gzip"
+      {{- if .Values.proxy.https }}
+      http_proxy_url = "${KOMOKW_HTTPS_PROXY_URL}"
+      {{- else if .Values.proxy.http }}
+      http_proxy_url = "${KOMOKW_HTTP_PROXY_URL}"
```
Comment (Noy-Simon) [inline charts/k8s-watcher/templates/configmap-daemon.yaml:25]: why else if with the same condition?

**Review by Noy-Simon — APPROVED** [review]

**Pull request merged**